services/logs2logs-db/Dockerfile

```diff
@@ -14,5 +16,9 @@ ENV PATH_CONFIG=null
 ENV RABBITMQ_HOST="rabbitmq" \
     RABBITMQ_USER="guest" \
     RABBITMQ_PASSWORD="guest" \
-    ELASTICSEARCH_URL="http://logs-db:9200"
+    ELASTICSEARCH_URL="http://logs-db:9200" \
+    LOGSTASH_USERNAME="$LOGSTASH_USERNAME" \
```

Comment:
we want to define the default here, not reuse already existing variables, so:
`LOGSTASH_USERNAME="username"` and `LOGSTASH_PASSWORD="password"`